index.html

```diff
@@ -708,6 +708,9 @@ <h3 class="col-md-11 col-lg-11">Electrify</h3>
   <!-- end footer -->
   <script src="./scripts/mobile-pillars.js"></script>
   <script src="https://code.jquery.com/jquery-3.1.1.js" integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA=" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
```

Comment:
why are we loading bootstrap at the bottom of the footer? there should be one load which I am assuming is happening already somewhere in the <head> tag?